drivers/SmartThings/zigbee-motion-sensor/src/ikea/init.lua

```diff
@@ -63,6 +63,7 @@ local function zdo_binding_table_handler(driver, device, zb_rx)
       return
     end
   end
+  driver:add_hub_to_zigbee_group(0x0000) -- fallback if no binding table entries found
```

Comment:
The DTH also added the device to group 0x0000 so I think we also need to do that here.

Reply:
you mean this line:

`cmds += zigbee.command(CLUSTER_GROUPS, 0x00, "${zigbee.swapEndianHex(zigbee.convertToHexString(groupAddr, 4))} 00")`

Reply:
Yep that's the one